syntax = "proto3";

package exonum.python.runtime;

// Python artifact spec.
message PythonArtifactSpec {
    // Name of the source wheel file.
    string source_wheel_name = 1;
    // Service library name (it will be used to import service module).
    string service_library_name = 2;
    // Name of the service class to be imported from the library.
    string service_class_name = 3;
    // Expected hash of the artifact.
    bytes hash = 4;
}
